pkg/clearsource/clearsource.go

```diff
@@ -64,7 +65,11 @@ func BuildFn(ctx *gcp.Context, exclusions []string) error {
 		ctx.Span("Clear source", now, gcp.StatusOk)
 	}(time.Now())
 
+	userExclusions := strings.Split(os.Getenv(env.ClearSourceExclude), ":")
```

Comment:
This should probably be wrapped in 

`if exclude := env.ClearSourceExclude); exclude != ""`

Otherwise userExclusions will always be a slice with one empty string if the env var is not specified.